api.go

```diff
@@ -53,6 +53,11 @@ func main() {
 
 	r.Handle("/sessions/{sessionId}/ws/", server)
 
+	// Reverse proxy
+	proxyHandler := handlers.NewMultipleHostReverseProxy()
+	r.Host(`{node}-{port:[0-9]*}.play-with-docker.com`).Handler(proxyHandler)
+	r.Host(`{node}.play-with-docker.com`).Handler(proxyHandler)
+
```

Comment:
We could have the base hostname in an environment variable? That would be the most straight-forward case IMO.